public/app/plugins/panel/graph/template.ts

```diff
@@ -11,7 +11,7 @@ const template = `
       items="ctrl.contextMenuCtrl.menuItems"
       onClose="ctrl.onContextMenuClose"
       getContextMenuSource="ctrl.contextMenuCtrl.getSource"
-      formatSourceDate="ctrl.dashboard.formatDate"
+      formatSourceDate="ctrl.formatDate"
```

Comment:
Thing is we would end up with relying on angular's scope anyways to control the origin of the context menu (i.e. the datapoint on the graph). So the context menu 'logic' would be distributed across GraphCtrl (to keep track of the origin) and react component controlling the visibility of the menu. I will try to apply data links in react graph panel next week and this will be an opportunity to align it here as well and get rid of angular directive.